pages/guideforms.py

```diff
@@ -642,6 +642,33 @@ def validate(self, value):
             'https://github.com/WICG/idle-detection/blob/main/HOWTO.md'
             '">Example 2</a>.')),
 
+    'dt_milestone_desktop_start': forms.IntegerField(
+        required=False, label='DevTrail on desktop',
+        widget=forms.NumberInput(attrs={'placeholder': 'Milestone #'}),
+        help_text=('First milestone that will allow developers to try out '
+                   'this feature on desktop platforms by setting a flag.')),
+
+    'dt_milestone_android_start': forms.IntegerField(
+        required=False, label='DevTrail on Android',
+        widget=forms.NumberInput(attrs={'placeholder': 'Milestone #'}),
+        help_text=('First milestone that will allow developers to try out '
+                   'this feature on iOS by setting a flag.  '
+                   'Set this only if different from desktop milestone.')),
```

Comment:
Are you autofilling based on desktop? If my feature is desktop only, how is this distinguished from desktop-applies elsewhere? Setting this should be explicit, even if it duplicates desktop.

Reply:
I was not planning on autofilling, but I see what you mean.  If I autofill the desktop value into other platform fields, then users could delete the value to indicate that the feature will not be available on that platform.  I'll try that.

Reply:
OK, I added JS for this and added some wording to explain what is happening.   Please take a look.

Reply:
>Set this only if different from desktop milestone.

I don't think this line should have gone away. This is what tells you that you can override. That may not be obvious.

Reply:
I thought that your original comment on this line of code called out two possible meanings for a missing platform-specific milestone:
desktop-applies: Missing milestone means that it is the same as the desktop milestone, or
desktop-only: Missing milestone means that the feature will not be available on that platform.
So, I implemented the JS code to auto-fill values so that the desktop-applies use case happens automatically, and the user can clear out one or more platform milestone field to indicate the desktop-only situation (or at least not available on a given platform).

Reply:
I'm going to go ahead and merge what I have, but I am certainly open to revising it further in a new PR.